webui/src/App.js

```diff
@@ -21,21 +21,40 @@ import {
   Link
 } from "react-router-dom";
 
+// Amount of panels.
+let panels = 5;
+let panel_indexes = init_array(panels, true);
+
 // Keep track of the current thresholds fetched from the backend.
 // Make it global since it's used by many components.
-let kCurThresholds = [0, 0, 0, 0];
+let kCurThresholds = init_array(panels);
 
 // A history of the past 'max_size' values fetched from the backend.
 // Used for plotting and displaying live values.
 // We use a cyclical array to save memory.
-let kCurValues = [[0, 0, 0, 0]]
+let kCurValues = [init_array(panels)];
 const max_size = 1000;
 let oldest = 0;
 
 var ws;
 const wsCallbacks = {};
 const wsQueue = [];
 
+function init_array(length, incremental = false) {
```

Comment:
I don't think we need this function either.

for an array of zeroes you can do:
```
new Array(num_panels).fill(0)
```

for a sequential list you can do:
```
[...Array(num_panels).keys()]
```

Reply:
Thanks! ^^"